NextDom/nextdom-core
# New statics themes

Modifs :
- Themes management simplification
- Themes are static now and not regenerate
- You can choose base (dark, light, mix) and identity (from 8colors)
- Preview of change is now a iframe dynamique and navigable

To test :
- Make a postinst
- Go on custom page
- Play with theme choices and preview, save and check all is good
